cmd/admin-router.go

```diff
@@ -156,6 +156,12 @@ func registerAdminRouter(router *mux.Router, enableConfigOps bool) {
 			HandlerFunc(gz(httpTraceHdrs(adminAPI.SetPolicyForUserOrGroup))).
 			Queries("policyName", "{policyName:.*}", "userOrGroup", "{userOrGroup:.*}", "isGroup", "{isGroup:true|false}")
 
+		// Attach policies to user
+		adminRouter.Methods(http.MethodPut).Path(adminVersion+"/attach-user-policies").HandlerFunc(gz(httpTraceHdrs(adminAPI.AttachPolicyToUser))).Queries("accessKey", "{accessKey:.*}", "policiesToAdd", "{policiesToAdd:.*}")
+
+		// Detach policies from user
+		adminRouter.Methods(http.MethodPut).Path(adminVersion+"/detach-user-policies").HandlerFunc(gz(httpTraceHdrs(adminAPI.DetachPolicyFromUser))).Queries("accessKey", "{accessKey:.*}", "policiesToDetach", "{policiesToDetach:.*}")
```

Comment:
See comments for group attach/detach.

Reply:
Also you can rename the query parameter `accessKey` here to just `user`. `accessKey` is going to have a different meaning as we are going to rename service accounts as accessKey.